koku/masu/processor/tasks.py

```diff
@@ -107,69 +163,56 @@ def get_report_files(
         f" provider: {provider_type}\n"
         f" provider_uuid: {provider_uuid}\n"
     )
-    for report in reports:
-        stmt += " file: " + str(report["file"]) + "\n"
-    LOG.info(stmt[:-1])
-    reports_to_summarize = []
-    start_date = None
-
-    for report_dict in reports:
-        with transaction.atomic():
-            try:
-                manifest_id = report_dict.get("manifest_id")
-                file_name = os.path.basename(report_dict.get("file"))
-                with ReportStatsDBAccessor(file_name, manifest_id) as stats:
-                    started_date = stats.get_last_started_datetime()
-                    completed_date = stats.get_last_completed_datetime()
-
-                # Skip processing if already in progress.
-                if started_date and not completed_date:
-                    expired_start_date = started_date + datetime.timedelta(
-                        hours=Config.REPORT_PROCESSING_TIMEOUT_HOURS
-                    )
-                    if DateAccessor().today_with_timezone("UTC") < expired_start_date:
-                        LOG.info(
-                            "Skipping processing task for %s since it was started at: %s.",
-                            file_name,
-                            str(started_date),
-                        )
-                        continue
-
-                stmt = (
-                    f"Processing starting:\n"
-                    f" schema_name: {customer_name}\n"
-                    f" provider: {provider_type}\n"
-                    f" provider_uuid: {provider_uuid}\n"
-                    f' file: {report_dict.get("file")}'
-                )
-                LOG.info(stmt)
-                if not start_date:
-                    start_date = report_dict.get("start_date")
-                worker_stats.PROCESS_REPORT_ATTEMPTS_COUNTER.labels(provider_type=provider_type).inc()
-                _process_report_file(schema_name, provider_type, provider_uuid, report_dict)
-                known_manifest_ids = [report.get("manifest_id") for report in reports_to_summarize]
-                if report_dict.get("manifest_id") not in known_manifest_ids:
-                    report_meta = {
-                        "schema_name": schema_name,
-                        "provider_type": provider_type,
-                        "provider_uuid": provider_uuid,
-                        "manifest_id": report_dict.get("manifest_id"),
-                    }
-                    reports_to_summarize.append(report_meta)
-            except (ReportProcessorError, ReportProcessorDBError) as processing_error:
-                worker_stats.PROCESS_REPORT_ERROR_COUNTER.labels(provider_type=provider_type).inc()
-                LOG.error(str(processing_error))
-                WorkerCache().remove_task_from_cache(cache_key)
-                raise processing_error
+    if report_dict:
+        stmt += " file: " + str(report_dict["file"]) + "\n"
+        LOG.info(stmt[:-1])
```

Comment:
```suggestion
        stmt += f" file: {report_dict['file']}"
        LOG.info(stmt)
```
this logging weirdness was necessary when we used the `for` loop.